src/main/java/com/codeborne/selenide/Selectors.java

```diff
@@ -62,7 +62,7 @@ public static By byText(String elementText) {
    * @return standard selenium By cssSelector criteria
    */
   public static By byAttribute(String attributeName, String attributeValue) {
-    return By.cssSelector(String.format("[%s='%s']", attributeName, attributeValue));
+    return By.cssSelector(String.format("[%s=\"%s\"]", attributeName, attributeValue));
```

Comment:
it doesn' solve the problem yet. You must also replace existing  "-s  and '-s  with \" and \'